docs/Changelog.md

```diff
@@ -80,6 +82,9 @@
   * Fix result type of projection (`.!`) - e.g. `#array.![#this]` returns a type array instead of a type List.
 * [#6685](https://github.com/TouK/nussknacker/pull/6685) Fixed an issue with dictionary parameter editor language being set to spel when no default value was present.
 * [#6797](https://github.com/TouK/nussknacker/pull/6797) [#6815](https://github.com/TouK/nussknacker/pull/6815) Performance optimization of Avro processing
+* [#6578](https://github.com/TouK/nussknacker/pull/6578) Introduced support for displaying a docsUrl with an icon in scenario properties. This feature is based on the processingType and is configurable via the model.conf file.
```

Comment:
Is it intentional? It was missing?

Reply:
yes and yes. It was not described in a changelog